src/components/InMarketAsset/InMarket.tsx

```diff
@@ -74,7 +81,8 @@ const InMarket = () => {
   const closeCard = () => {
     setOpenModal(false)
   }
-
+ // if user try to enter wrong id
+ if (!nft) return null
```

Comment:
should be redirect to inventory if !nft.

Reply:
![image](https://user-images.githubusercontent.com/23531140/132316531-850320bb-954e-48ee-99fc-21eb9664f828.png)
I did it in useEffect otherwise react  will raise a warning